cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/HttpRequestBuilderImpl.java

```diff
@@ -32,21 +32,30 @@
 import java.util.Map;
 import java.util.Objects;
 import java.util.Properties;
+import mil.army.usace.hec.cwms.http.client.request.HttpPostRequest;
+import mil.army.usace.hec.cwms.http.client.request.HttpRequestExecutor;
+import mil.army.usace.hec.cwms.http.client.request.HttpRequestMediaType;
+import mil.army.usace.hec.cwms.http.client.request.HttpRequestMethod;
 import okhttp3.HttpUrl;
+import okhttp3.MediaType;
 import okhttp3.OkHttpClient;
 import okhttp3.Request;
+import okhttp3.RequestBody;
 import okhttp3.Response;
 import okhttp3.ResponseBody;
 import usace.metrics.noop.NoOpTimer;
 import usace.metrics.services.Metrics;
 import usace.metrics.services.Timer;
 
-public class HttpRequestBuilderImpl implements HttpRequestBuilder {
+public final class HttpRequestBuilderImpl implements HttpRequestBuilder {
```

Comment:
You added the final back in? Does Cumulus not need to extend it?

Reply:
Hmm, something might've gotten messed up on the rebase I did. Did you make other changes in the commit where you updated that?

Reply:
I found the file. I'll do a diff on that, but I agree that it shouldn't be final anymore. I'll update it.